app/controllers/users_controller.rb

```diff
@@ -1,2 +1,7 @@
 class UsersController < ApplicationController
+
+  def show
+    @games = Game.all
```

Comment:
Shouldn't we have this index in one place? We have the same functionality of `@games = Game.all` in the games_controller under `def index`